roles/kafka_controller/tasks/health_check.yml

```diff
@@ -0,0 +1,44 @@
+---
+# health check for kafka controller
+- name: Add controller listener in client properties
+  shell: |
+    echo confluent.use.controller.listener=true >> {{kafka_controller.client_config_file}}
```

Comment:
so every time the health check runs, a new line gets added to this file? I'd look for a way around that